src/simple-cache-client.ts

```diff
@@ -984,7 +985,25 @@ export class SimpleCacheClient {
     return await client.sortedSetGetScore(cacheName, sortedSetName, value);
   }
 
-  // sorted set get scores
+  /**
+   * Look up the scores of multiple elements in the sorted set, by the value of the elements.
+   *
+   * @param {string} cacheName - The cache containing the sorted set.
+   * @param {string} sortedSetName - The sorted set to fetch from.
+   * @param {(string | Uint8Array)[]} values - The values of the elements whose scores we are retrieving.
+   * @returns {Promise<CacheSortedSetGetScores.Response>}
+   * {@link CacheSortedGetScores.Hit} containing the scores of the requested elements when found.
+   * {@link CacheSortedGetScores.Miss} when the element or collection does not exist.
+   * {@link CacheSortedGetScores.Error} on failure.
+   */
+  public async sortedSetGetScores(
+    cacheName: string,
+    sortedSetName: string,
+    values: (string | Uint8Array)[]
```

Comment:
in most places we have been doing `string[] | Uint8Array[]`, which I think is different from this; we should probably stick to the other one?

Reply:
I prefer the broader one, though it may be confusing unless we change the rest. So we can change to the more restricted one.